bot/exts/moderation/incidents.py

```diff
@@ -180,18 +277,33 @@ class Incidents(Cog):
         * If `Signal.ACTIONED` or `Signal.NOT_ACTIONED` were chosen, attempt to
           relay the incident message to #incidents-archive
         * If relay successful, delete original message
+        * Delete quotation message if cached
         * See: `on_raw_reaction_add`
 
     Please refer to function docstrings for implementation details.
     """
 
+    # This dictionary maps an incident report message to the message link embed's ID
+    # RedisCache[discord.Message.id, discord.Message.id]
+    message_link_embeds_cache = RedisCache()
+
     def __init__(self, bot: Bot) -> None:
         """Prepare `event_lock` and schedule `crawl_task` on start-up."""
         self.bot = bot
 
+        self.bot.loop.create_task(self.get_webhook())
+
         self.event_lock = asyncio.Lock()
         self.crawl_task = self.bot.loop.create_task(self.crawl_incidents())
 
+    async def get_webhook(self) -> None:
+        """Fetch and store message link embeds webhook, present in #incidents channel."""
```

Comment:
Did you mean `embed's`? Is there a missing `the` here? This docstring is a bit confusing.